middleware.js

```diff
@@ -4,8 +4,7 @@ import App from './app/app'
 
 module.exports = (options) =>
   (req, res, next) => {
-    const endpointRegExp = new RegExp(options.middlewarePath, 'i')
-    if (endpointRegExp.test(req.url)) {
+    if (req.url.match(new RegExp(options.middlewarePath, 'i'))) {
```

Comment:
tbf I was thinking to have even something simpler such as 

```javascript
if(req.url === options.middlewarePath)
```
and maybe we do not need to think about moving the instantiation of the RegExp somewhere else; what do you think?